'Exception handing'

# def fun(a,b,c):

#     try:
#         print help(chr())
#         return a/b,"chr :",chr(c)

#     except Exception as k:
#         print "Exception :",Exception
#         return "k :", k,"chr :",dir(chr(c))

# print fun(10,1,11)

'Example2'

def fun(a,b):

    try:
        print "number",a+b

    except Exception as e:
        print e

print (fun('anji',6))


        